use-cases/managed-connectors-integration/README.md

```diff
@@ -0,0 +1,25 @@
+# Managed Connectors Integration
```

Comment:
Can you please add a prerequisite section with link to required tools? As far as I see it requires rhoas and kcat CLI